crates/module-system/module-implementations/sov-evm/src/config.rs

```diff
@@ -46,7 +46,7 @@ impl Default for EvmChainSpec {
             coinbase: Address::ZERO,
             block_gas_limit: ETHEREUM_BLOCK_GAS_LIMIT_30M,
             block_timestamp_delta: SLOT_DURATION.as_secs(),
-            base_fee_params: alloy_eips::eip1559::BaseFeeParams::ethereum(),
+            base_fee_params: alloy_eips::eip1559::BaseFeeParams::new(1, 1),
```

Comment:
I will update #1510 to ensure we initialize all the EVM parameters correctly (these values are not set yet properly).